llvm/test/MC/RISCV/pr30955.s

```diff
@@ -0,0 +1,29 @@
+// RUN: llvm-mc -triple riscv32 %s -o - | FileCheck %s --check-prefix=ASM
```

Comment:
Best to use a descriptive name instead of `PRxxxxx`.

Reply:
align-zero.s might be a good name

Reply:
I chose `align-fill-byte-zero.s` to show that it's really about byte alignment directives, and not wider alignment directives (`.{b,p2,}align{l,w}`)

Reply:
Sounds great!